subgraph/src/mappings/request-manager.ts

```diff
@@ -0,0 +1,147 @@
+import { BigInt, Bytes } from "@graphprotocol/graph-ts";
+import {
+  ClaimMade, ClaimStakeWithdrawn,
+  DepositWithdrawn,
+  FillInvalidatedResolved,
+  OwnershipTransferred,
+  RequestCreated,
+  RequestResolved
+} from "../../generated/RequestManager/RequestManager";
+import {Claim, FillState, Request} from '../../generated/schema';
+import {ADDRESS_ZERO_BYTES, ZERO_BI, ZERO_BYTES, CLAIM_ID_EXPIRED_REQUEST} from '../utils/constants';
+import { loadTransaction } from "../utils";
+
+export function handleClaimMade(event: ClaimMade): void {
+  const claim = new Claim(event.params.claimId.toString());
+
+  claim.claimer = event.params.claimer;
+  claim.claimerStake = event.params.claimerStake;
+  claim.lastChallenger = event.params.lastChallenger;
+  claim.challengerStakeTotal = event.params.challengerStakeTotal;
+  claim.termination = event.params.termination;
+  claim.fillId = event.params.fillId;
+  claim.transaction = loadTransaction(event).id
+  claim.emittedAt = event.block.timestamp
+  claim.emittedByTx = event.transaction.hash
+  claim.contractAddress = event.address
+  claim.resolved = false;
+
+  const request = Request.load(event.params.requestId);
+
+  if (request) {
+    claim.request = request.id;
+    claim.save();
+
+    const claims = request.claims;
+    request.claims = claims && claims.length > 0 ? claims.concat([claim.id]) : [claim.id];
```

Comment:
Yes, apparently what you are saving in such definitions is the object id. I also thought that when I define the schema as `Claim!` that I would have to pass actual objects, but you need ids there. The graphQL server later makes sure to properly load the ids for objects. 
The generated typescript was pointing this out and also if you try to build the graph with an object there it will fail.